src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -429,6 +450,47 @@ void AppHost::Close()
     }
 }
 
+void AppHost::_revokeWindowCallbacks()
+{
+    _window->MouseScrolled(_windowCallbacks.MouseScrolled);
+    _window->WindowActivated(_windowCallbacks.WindowActivated);
+    _window->WindowMoved(_windowCallbacks.WindowMoved);
+    _window->ShouldExitFullscreen(_windowCallbacks.ShouldExitFullscreen);
+    _window->WindowCloseButtonClicked(_windowCallbacks.WindowCloseButtonClicked);
+    _window->DragRegionClicked(_windowCallbacks.DragRegionClicked);
+    _window->WindowVisibilityChanged(_windowCallbacks.WindowVisibilityChanged);
+    _window->UpdateSettingsRequested(_windowCallbacks.UpdateSettingsRequested);
+    _window->MaximizeChanged(_windowCallbacks.MaximizeChanged);
+    _window->AutomaticShutdownRequested(_windowCallbacks.AutomaticShutdownRequested);

```

Comment:
au to re vo ke rs!

Reply:
UGH IT ISNT A WINRT TYPE STOP YOU'RE HURTING ME

Reply:
![image](https://i.giphy.com/media/RbSmVaVGptW03Wjw3a/giphy.webp)

Reply:
Oh my GOD i almost left the same exact comment again, someone pls maim me